libqtile/widget/google_calendar.py

```diff
@@ -62,14 +65,25 @@
 import re
 import dateutil.parser
 import threading
+import os
 
-from apiclient.discovery import build
-from oauth2client.client import OAuth2WebServerFlow
-from oauth2client.tools import run
-import oauth2client.file
+from apiclient import discovery
+import oauth2client
+from oauth2client import client
+from oauth2client import tools
```

Comment:
Can we merge both lines together like `from oauth2client import client, tools`
Otherwise nice work :)